### PR TITLE
The-effects-of-loneliness

### LINKED ISSUE #325: 📝[Docs]: The effects of loneliness on health and strategies for recovery.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Description

Loneliness “can have serious mental and physical complications that worsen if ignored.” She added that, “social isolation and loneliness lead to higher risk of high blood pressure, heart disease, obesity, anxiety, depression, memory issues and even death.”

### Suggested Change

Loneliness can leave people feeling isolated and disconnected from others. It is a complex state of mind that can be caused by life changes, mental health conditions, poor self-esteem, and personality traits. Loneliness can also have serious health consequences including decreased mental wellness and physical problems.

### Rationale

_No response_

### Urgency

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by Aditijainnn**
#### PR Description
Added The-effects-of-loneliness-on-health-and-strategies-for-recovery  #325

#### Type of Change
- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [x] Documentation update

#### Select your work-area
- [ ] Frontend
- [ ] Backend
- [x] Documentation
- [ ] Others

#### Fixes (mention the issue number which this fixes)
#closes
 #325

#### Checklist
- [x] I have updated my branch and synced it with the project's 'develop' branch before making this PR.
- [x] I have optimized the file changes.
- [x] I have added a snapshot of my work example.
- [x] I have made a PR to the project's develop branch.

#### Undertaking
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] I have checked for plagiarism and assure its authenticity.
- [x] I have read and followed the code of conduct for this repository. I understand that violation of this undertaking may have legal consequences.

- [x] I Agree

### CODE REVIEW

File: DataSources/The-effects-of-loneliness-on-health-and-strategies-for-recovery/Article-english.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    a 50% increased risk of dementia
+
+ResearchTrusted Source suggests that in people with heart failure, loneliness can:
+
+    increase the risk of death by almost four times
+    cause a 68% increased risk of hospitalization
+    cause a 57% increased risk of visits to the emergency department
+
+
+## Ways of managing loneliness
```
Comment (SB2318) [inline DataSources/The-effects-of-loneliness-on-health-and-strategies-for-recovery/Article-english.md:167]: Expand this thing more. with each point, please add some more details

**Review by SB2318 — COMMENTED** [review]
Good work!

File: DataSources/The-effects-of-loneliness-on-health-and-strategies-for-recovery/Article-english.md (hunk 1 above)
Comment (Aditijainnn) [reply DataSources/The-effects-of-loneliness-on-health-and-strategies-for-recovery/Article-english.md:167]: > Expand this thing more. with each point, please add some more details

ok

**Comment by Aditijainnn** [comment]
> Good work!

done changes

**Review by SB2318 — COMMENTED** [review]
Please update your branch.
image

**Pull request merged**